front_end/panels/ai_chat/ui/settings/providerConfigs.ts

```
@@ -0,0 +1,145 @@
+// found in the LICENSE file.
+
+import { i18nString, UIStrings } from './i18n-strings.js';
+import type { ProviderConfig } from './providers/GenericProviderSettings.js';
+import {
+  OPENAI_API_KEY_STORAGE_KEY,
+  BROWSEROPERATOR_API_KEY_STORAGE_KEY,
+  GROQ_API_KEY_STORAGE_KEY,
+  CEREBRAS_API_KEY_STORAGE_KEY,
+  ANTHROPIC_API_KEY_STORAGE_KEY,
+  GOOGLEAI_API_KEY_STORAGE_KEY,
+} from './constants.js';
```

ESLint

[error] `./constants.js` import should occur before import of `./i18n-strings.js`

(import/order)